.rubocop_todo.yml

```diff
@@ -14,3 +14,9 @@ Layout/SpaceAroundOperators:
   Exclude:
     - 'lib/jekyll/commands/build.rb'
     - 'test/test_configuration.rb'
+
+# Offense count: 1
+# Cop supports --auto-correct.
+Lint/RedundantCopDisableDirective:
+  Exclude:
+    - 'lib/jekyll/document.rb'
```

Comment:
If RuboCop thinks it is redundant, why are we not removing the directive?

Reply:
fixed, thanks